app/models/review.rb

```diff
@@ -0,0 +1,11 @@
+class Review < ActiveRecord::Base
+  belongs_to :bar
+  belongs_to :user
+
+  validates :title, presence: true
+  validates :body, presence: true
+  validates :bar_id, presence: true
+  validates :user_id, presence: true
+  validates :rating, presence: true,
+  numericality: { integer: true }, inclusion: { in: 1..5 }
```

Comment:
I'd say the norm is for this should read something like:

```
validates :rating, 
  presence: true,
  numericality: { integer: true }, 
  inclusion: { in: 1..5 }
```

(Pretty sure that's what Hound is looking for!)